spec/factories/contributors.rb

```diff
@@ -5,9 +5,7 @@
     first_name { 'John' }
     last_name { 'Doe' }
     data_processing_consent { true }
-    sequence :email do |n|
-      "contributor#{n}@example.org"
-    end
+    email { Faker::Internet.email }
```

Comment:
this change is related to this PR since the seed rake task fails when trying to create simply sequential emails.